contracts/test/P256Account.t.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/Counter.sol";
+import "../src/core/EntryPoint.sol";
+import "../src/P256Account.sol";
+import "../src/P256AccountFactory.sol";
+import {UserOperation} from "../src/interfaces/UserOperation.sol";
+
+/**
+ * @title CounterTest
+ * @author richard@fun.xyz
+ * @notice This is a sanity test for the account function.
+ * We want to be able to send a userOp through the entrypoint and have it execute
+ */
+contract CounterTest is Test {
+    Counter public counter;
+    EntryPoint public entryPoint;
+    P256AccountFactory public accountFactory;
+    P256Account public account;
+
+    // -------------------- 🧑‍🍼 Account Creation Constants 🧑‍🍼 --------------------
+    bytes constant publicKey = "iliketturtles";
```

Comment:
this will be fixed when i add instances to the circuit xD

Reply:
Sg, no changes needed here right?